Tests/test_file_xvthumb.py

```diff
@@ -1,13 +1,35 @@
-from helper import unittest, PillowTestCase
+from helper import hopper, unittest, PillowTestCase
 
-from PIL import XVThumbImagePlugin
+from PIL import Image, XVThumbImagePlugin
+
+TEST_FILE = "Tests/images/hopper.p7"
 
 
 class TestFileXVThumb(PillowTestCase):
 
+    def test_open(self):
+        # Act
+        im = Image.open(TEST_FILE)
+
+        # Assert
+        self.assertEqual(im.format, "XVThumb")
+        self.assert_image_similar(im, hopper("P"), 49)
```

Comment:
Here's what `im.convert("RGB")` looks like.
It has epsilon of 42.0398 against `hopper("RGB")`:
![image](https://cloud.githubusercontent.com/assets/1324225/23612406/0615ac3c-0284-11e7-8452-9a63344c621f.png)

How do we quantise it to funky palette?